src/main/webapp/js/instructor.js

```diff
@@ -434,3 +434,35 @@ function updateHoverShowPictureEvents(actualLink, resolvedLink) {
 }
 
 // --------------------------------------------------------------------------
+
+/**
+ * Selects contents inside an element.
+ * @param elementNode a HTML DOM element object, not a jQuery object
```

Comment:
Hm you could if you want to, I'm not too sure about the preferences for this. Maybe @wkurniawan07 could comment on this :p